nodejs/src/access_token_common.js

```diff
@@ -1,19 +1,18 @@
-import crypto from 'crypto'
+import crypto from 'crypto';
 import fs from 'fs';
-import path from 'path'
+import path from 'path';
 
-import {ApiCommon} from './api_common.js'
+import { ApiCommon } from './api_common.js';
 
 export class AccessTokenCommon extends ApiCommon {
-
-  constructor(api_config, {name}) {
+  constructor(api_config, { name }) {
     super(api_config);
 
     const auth = `${api_config.app_id}:${api_config.app_secret}`;
     const b64auth = Buffer.from(auth).toString('base64');
     this.api_uri = api_config.api_uri;
-    this.auth_headers = {Authorization: `Basic ${b64auth}`}
-    this.name = (name) ? name : `access_token_${api_config.version}`;
+    this.auth_headers = { Authorization: `Basic ${b64auth}` };
+    this.name = (name) || `access_token_${api_config.version}`;
```

Comment:
nit: you can remove the () e.g name || `access_token_${api_config.version}`;

Reply:
Good point. Let me see if there's an eslint rule that will check for extra parentheses.

Reply:
Added, autofixed, and pushed the eslint no-extra-parens rule.